website/docs/concepts/terminology.md

```diff
@@ -75,7 +75,7 @@ An Interpreter is a process that runs during [Synthesis Time](#synthesis-time) t
 
 Examples include:
 
-- [Amazon States Language](../api/classes/ASL.md)
+- [Amazon States Language](../api/classes/ASL-1.md)
```

Comment:
Why did this change

Reply:
I am not sure, version of the typedoc was upgraded, something must have changed.